packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -1130,6 +1141,7 @@ export class Virtualizer<T extends object, V, W> {
     }
   }
 
+  // TODO: perhaps have the isTestEnv check here as well and set to Infinity/Infinity? Or just rely on ScrollView setting contentSize as infnity?
```

Comment:
We can't set a default contentSize when `isTestEnv` is true, because this is used by different components that would have different sizes, dropdown, verse table, verse card layout. right?

Reply:
this is more just uncertainty about what level to do the Infinity width/height mocking since this will be set by ScrollView. Perhaps setting it here is more straightforward, not sure of the other ramifications though.